frontend/awx/resources/templates/TemplatePage/TemplatePage.tsx

```diff
@@ -36,9 +36,9 @@ export function TemplatePage() {
     error: isNotifAdminError,
     refresh: refreshNotifAdmin,
     isLoading: isNotifAdminLoading,
-  } = useGet<AwxItemsResponse<Organization>>('/api/v2/organizations', {
-    role_level: 'notification_admin_role',
-  });
+  } = useGet<AwxItemsResponse<Organization>>(
+    '/api/v2/organizations/?role_level=notification_admin_role'
```

Comment:
hmm, this should have worked the way it was. Is there a bug in `useGet`? (or maybe the problem was the missing trailing slash?). You don't necessarily need to change it back... we do it this way in other places, it looks like

Reply:
@keithjgrant and I chatted more about this 1:1. I checked locally and useGet doesn't handle 301 redirects when there is a trailing missing slash. This is an enhancement we should look into at a later time.